packages/core/src/components/dropdown/dropdown.tsx

```diff
@@ -99,6 +99,12 @@ export class Dropdown {
     alignmentAxis?: number;
   };
 
+  /**
+   * Define event to open element
+   * @since 1.3.0
+   */
+  @Prop() triggerEvent: 'click' | 'mouseover' = 'click';
```

Comment:
We need support for one or multiple custom event triggers (e.g. pseudo code: event | event[]).